uber/templates/registration/merch.html

```diff
@@ -3,6 +3,7 @@
 {% block content %}
 
 <script type="text/javascript">
+    {% set detect_barcode_scan = True %}
```

Comment:
Now merch can boop badges too! We probably don't have the equipment for it, though.

Note that the merch page has multiple 'badge_num' fields, so the volunteer will have to focus the 'correct' one for the scanning to work.